README.md

```diff
@@ -153,3 +153,56 @@ class MyComponent extends Component {
 }
 ```
 
+## Testing using React Test Utilities
+
+[lib/react_test_utils.dart](lib/react_test_utils.dart) is a Dart wrapper for the [React TestUtils](http://facebook.github.io/react/docs/test-utils.html) library allowing for tests to be made for React components in Dart.
+
+Here is an example of how to use React TestUtils within a Dart test.
+
+```dart
+import 'package:unittest/unittest.dart';
+import 'package:react/react.dart' as react;
+import 'package:react/react_client.dart' as reactClient;
+import 'package:react/react_test_utils.dart' as reactTestUtils;
+
+class MyTestComponent extends react.Component {
+  getInitialState() => {'text': 'testing...'};
+  render() {
+    return react.div({}, [
+        react.button({'onClick': (e) => setState({'text': 'success'})}),
+        react.span({'className': 'spanText'}, state['text'])
+    ]);
+  }
+}
+
+var myTestComponent = react.registerComponent(() => new MyTestComponent());
+
+void main() {
+  reactClient.setClientConfiguration();
+  var component = reactTestUtils.renderIntoDocument(myTestComponent({}));
```

Comment:
This name is actually misleading, as it renders into a detached DOM node.

_From https://github.com/facebook/react/blob/v0.12.1/src/test/ReactTestUtils.js#L41:_
>```js
>  renderIntoDocument: function(instance) {
>    var div = document.createElement('div');
>    // None of our tests actually require attaching the container to the
>    // DOM, and doing so creates a mess that we rely on test isolation to
>    // clean up, so we're going to stop honoring the name of this method
>    // (and probably rename it eventually) if no problems arise.
>    // document.documentElement.appendChild(div);
>    return React.render(instance, div);
>  },
>```